### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by OrielBanne**
checking with Assaf!

### CODE REVIEW

File: .ipynb_checkpoints/Untitled-checkpoint.ipynb [hunk 1]
```diff
@@ -0,0 +1,371 @@
+{
```
Comment (Snover98) [inline .ipynb_checkpoints/Untitled-checkpoint.ipynb:1]: remove this file

**Review by Snover98 — CHANGES_REQUESTED** [review]

File: SimpleNetColorInference.py [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+Tot_Loss = []
+Accuracy = []
+MaxEpochs = 50
+for epoch in range(MaxEpochs):  # loop over the dataset multiple times
+
+    epoch_loss = 0
+    running_loss = 0.0
+
+    # TODO: Dynamic Learning Rate
+    # adjust_learning_rate(optimizer, epoch)
```
Comment (Snover98) [inline SimpleNetColorInference.py:172]: why is this here?

**Comment by OrielBanne** [comment]
closed for train
